src/xunit.performance.api/PerformanceTestConfig.cs

```diff
@@ -0,0 +1,18 @@
+
+
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public class PerformanceTestConfig
+    {
+        public PerformanceTestConfig() 
+        {
+            iterations = 30;
+            timeout = 20000;
+        }
+
+        public int iterations;
+        public int timeout;
```

Comment:
For timeout:

1. Is this a total timeout for all iterations or a per-iteration timeout?  It should be clear based on the name.
2. Rather than using an int, which doesn't tell the user what units are used, I'd recommend a TimeSpan here.  It handles all the complexity for you.